develop/security-best-practices.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Security best practices
+description: Image security best practices guide
+keywords: docker, images, containers, vulnerability, cve
+redirect_from:
+  - /develop/scan-images/
+---
+
+As a developer, you can take a few simple steps to improve the security of your
+container. This includes:
+
+1. Choosing the right base image from a trusted source and keeping it small
+2. Using multi-stage builds
+3. Rebuilding images
+4. Check your image for vulnerabilities
+
+Now, let’s take a detailed look at each of these best practices:
```

Comment:
@dvdksn Everywhere else I see "you" or no pronouns, but here "let's". I am new and might be missing some style rule that's in place, but it feels somewhat out of place.